lib/wrappers/python/contourwall.py

```diff
@@ -137,7 +138,7 @@ def show(self, sleep_ms:int=0, optimize:bool=True) -> None:
         ```
         This example code will show the current state of the pixel array on the ContourWall.
         """
-        
+        self.pixels[:] = np.clip(self.pixels[:] * brightness, 0, 255)
         ptr: ctypes._Pointer[c_uint8] = ctypes.cast(ctypes.c_char_p(self.pixels.tobytes()), ctypes.POINTER(ctypes.c_uint8))
```

Comment:
The `np.clip` is not nescecary.

You can replace `self.pixels[:] * brightness` with `self.pixels[:] // (1/brightness)`. The `//` operator is just a divide, but it floors result of the expression. [Demo of `//`](https://www.w3schools.com/python/trypython.asp?filename=demo_oper_floordiv)